en/config_fw/trimming_guide_fixedwing.md

```diff
@@ -31,12 +31,26 @@ The correct order to set the above parameters is:
 1. Fly in stabilized mode at cruise speed and set the pitch setpoint offset (`FW_PSP_OFF`) to desired angle of attack.
    The required angle of attack at cruise speed corresponds to the pitch angle that the airplane needs to fly at in order to keep constant altitude during wing-leveled flight.
    If you are using an airspeed sensor, also set the correct cruise airspeed (`FW_AIRSPD_TRIM`).
-1. Look at the actuator controls in the log file (upload it to [Flight Review](https://logs.px4.io) and check the _Actuator Controls_ plot for example) and set the pitch trim (`TRIM_PITCH`).
+1. Run [auto-trimming](#auto-trimming) during a flight to set `TRIM_ROLL`, `TRIM_PITCH` and `TRIM_YAW`.
+
+   You could also manually set the pitch trim (`TRIM_PITCH`) by looking at the actuator controls in the log file (upload it to [Flight Review](https://logs.px4.io) and check the _Actuator Controls_ plot for example).
    Set that value to the average offset of the pitch signal during wing-leveled flight.
 
 Step 3 can be performed before step 2 if you don't want to have to look at the log, or if you feel comfortable flying in manual mode.
 You can then trim your remote (with the trim switches) and report the values to `TRIM_PITCH` (and remove the trims from your transmitter) or update `TRIM_PITCH` directly during flight via telemetry and QGC.
 
+### Auto-trimming
```

Comment:
Can calibration fail or result in bad values? If so, what would the user see? How would they know that they need to do it again?

Reply:
Except if there is a bug, normally it can't give bad values. If the trim values are not perfect, it will anyway correct them in the next flight. A user will feel bad trim values if he flies manually (he'll need to counteract the effect).
One can always reset the trims by setting the `TRIM_XXX` back to 0 and `FW_ATRIM_MODE` to "Calibration.